### ISSUE TITLE
Is it possible to add floating buttons of material design

### ISSUE THREAD

**Issue by F-Tash** [creativetimofficial/ct-vue-material-dashboard-pro#66]
Hello,
Is it possible to add material design floating buttons with with additional buttons like in picture
https://goo.gl/o248Nt
So when clicking (+) it should display additional buttons vertically with transition animation?

**Comment by dragosct10** [comment]
Hi! This product it's built on top of the vuematerial framework, so every button that you can use it can be found here https://vuematerial.io/components/button. 

Regards,
Dragos

**Comment by F-Tash** [comment]
Thanks for responding!
I found the 'FAB with speed dial' buttons are implimented in another vue material project: 
https://vuetifyjs.com/en/components/floating-action-buttons#fab-with-speed-dial
I found you implemented as well this theme based above project:
https://github.com/creativetimofficial/vuetify-material-dashboard
But could not found Pro version of this theme.
So is it possible somehow add this functionality to current pro version?
